dvl-a50/mavlink2resthelper.py

```diff
@@ -166,24 +169,29 @@ def __init__(self):
 }}
 """
 
-    def get_float(self, path: str) -> float:
+    def get_float(self, path: str, vehicle: int=None, component: int=None) -> float:
```

Comment:
The type here is wrong, should not be int, but Optional[int]

Reply:
Should we also change the return signature of `get` to be something that allows float or bool?

Reply:
We should choose between returning an optional over an float/nan.
Bool does not make a lot of sense to me.

Reply:
Sorry, I meant `str` or `bool`, since that's what it actually returns (this is for `get` - not `get_float`).